modules/doi/includes/utilities.inc

```diff
@@ -319,17 +319,19 @@ function islandora_doi_crossref_translator($crossref_data) {
               foreach ($child->childNodes as $grandchild) {
                 switch ($grandchild->nodeName) {
                   case 'title':
-                    $article_title = $mods->importNode($grandchild, TRUE);
-                    $titleinfo->appendChild($article_title);
+                    if (!is_null($grandchild)) {
```

Comment:
For consistency with [this line](https://github.com/Islandora/islandora_scholar/pull/288/files#diff-d342542a4ca9df72fe6804cad9aa55dbR260) in `islandora_scholar.drush.inc`, as well as [this](https://github.com/Islandora/islandora_scholar/pull/288/files#diff-52bb712f031b7c5151fc17f7508f7213R330) and [this](https://github.com/Islandora/islandora_scholar/pull/288/files#diff-52bb712f031b7c5151fc17f7508f7213R352) line in `modules/doi/includes/utilities.inc`, perhaps [that line](https://github.com/Islandora/islandora_scholar/pull/288/files#diff-52bb712f031b7c5151fc17f7508f7213R322) should also read
```php
                    if (!is_null($grandchild->firstChild)) {
```
(i.e., we only perform the translation if the node is non-empty)?